### PR TITLE
BC-5426 - Add a delete option to the school specific Privacy Policy and Terms of Use

### PR DESCRIPTION
**Pull request by davwas**
# Short Description

QA note:
Change tested on env: 
https://nbc-bc-5426-delete-policy-terms.cd.dbildungscloud.dev/
https://brb-bc-5426-delete-policy-terms.cd.dbildungscloud.dev/
https://default-bc-5426-delete-policy-terms.cd.dbildungscloud.dev/
No issues found. All accept criteria done. 
See JIRA ticket https://ticketsystem.dbildungscloud.de/browse/BC-5426

## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-5426

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

**Review by blazejpass — APPROVED** [review]

**Review by blazejpass — APPROVED** [review]

**Review by odalys-dataport — CHANGES_REQUESTED** [review]

File: src/components/organisms/administration/SchoolPolicy.vue [hunk 1]
```diff
@@ -34,7 +34,7 @@
 							{{
 								t("pages.administration.school.index.schoolPolicy.uploadedOn", {
 									date: dayjs(privacyPolicy.publishedAt).format(
-										t("format.dateTime")
+										t("format.date")
```
[the remark this answers is not in the document]
Comment (davwas) [reply src/components/organisms/administration/SchoolPolicy.vue]: fixed

File: src/components/organisms/administration/SchoolPolicy.unit.ts [hunk 2]
```diff
@@ -167,6 +171,16 @@ describe("SchoolPolicy", () => {
 		});
 	});
 
+	describe("when user clicks delete button", () => {
+		it("should change isDeletePolicyDialogOpen to true", () => {
+			const wrapper = setup();
+
+			expect((wrapper.vm as any).isDeletePolicyDialogOpen).toBe(false);
```
[the remark this answers is not in the document]
Comment (davwas) [reply src/components/organisms/administration/SchoolPolicy.unit.ts]: I've marked wrapper as any, apparently there is no easy way of using vue-test-utils with typescript: https://github.com/vuejs/vue-test-utils/issues/255
I haven't seen it solved differently in our app either.

File: src/components/organisms/administration/SchoolTerms.unit.ts [hunk 3]
```diff
@@ -167,6 +171,16 @@ describe("SchoolTerms", () => {
 		});
 	});
 
+	describe("when user clicks delete button", () => {
+		it("should change isDeleteTermsDialogOpen to true", () => {
+			const wrapper = setup();
+
+			expect((wrapper.vm as any).isDeleteTermsDialogOpen).toBe(false);
```
[the remark this answers is not in the document]
Comment (davwas) [reply src/components/organisms/administration/SchoolTerms.unit.ts]: I've marked wrapper as any, apparently there is no easy way of using vue-test-utils with typescript: https://github.com/vuejs/vue-test-utils/issues/255
I haven't seen it solved differently in our app either.

**Review by blazejpass — DISMISSED** [review]

**Review by blazejpass — APPROVED** [review]

**Review by blazejpass — APPROVED** [review]

File: src/components/organisms/administration/SchoolPolicy.vue [hunk 4]
```diff
@@ -42,9 +42,7 @@
 						<template v-if="privacyPolicy">
 							{{
 								t("pages.administration.school.index.schoolPolicy.uploadedOn", {
-									date: dayjs(privacyPolicy.publishedAt).format(
-										t("format.date")
-									),
+									date: dayjs(privacyPolicy.publishedAt).format("DD.MM.YYYY"),
```
Comment (odalys-dataport) [inline src/components/organisms/administration/SchoolPolicy.vue]: Please use DATETIME_FORMAT exported from datetime plugin

**Review by odalys-dataport — CHANGES_REQUESTED** [review]

File: src/components/organisms/administration/SchoolPolicy.vue (hunk 4 above)
Comment (davwas) [reply src/components/organisms/administration/SchoolPolicy.vue]: I think you have commented on the outdated file :)
The newest version uses different date displaying method.

**Review by odalys-dataport — APPROVED** [review]

**Pull request merged**